src/shared/components/post/post-form.tsx

```diff
@@ -264,25 +264,16 @@ export class PostForm extends Component<PostFormProps, PostFormState> {
         },
         communitySearchOptions: [selectedCommunityChoice].concat(
           (
-            this.props.initialCommunities?.map(
-              ({ community: { id, title } }) => ({
-                label: title,
-                value: id.toString(),
-              }),
-            ) ?? []
+            this.props.initialCommunities?.map(cv => communityToChoice(cv)) ??
```

Comment:
Nitpick: This can be simplified somewhat:

```suggestion
            this.props.initialCommunities?.map(communityToChoice) ??
```

Reply:
Good catch, thx.